packages/cmrjs/cmr-utils.js

```diff
@@ -633,12 +671,14 @@ async function updateEcho10XMLMetadata({
     'AssociatedBrowseImageUrls.ProviderBrowseUrl', []));
 
   const removedURLs = onlineAccessURLsToRemove(files, buckets);
-  const newURLs = constructOnlineAccessUrls({
+  let newURLs = await constructOnlineAccessUrls({
     files,
     distEndpoint,
     buckets,
-    cmrGranuleUrlType
-  }).concat(getS3CredentialsObject(urljoin(distEndpoint, s3CredsEndpoint)));
+    cmrGranuleUrlType,
+    distributionBucketMap
+  });
+  newURLs = newURLs.concat(getS3CredentialsObject(urljoin(distEndpoint, s3CredsEndpoint)));
```

Comment:
Since `getS3CredentialsObject` returns an object, this should probably be `push`, not `concat`.

Reply:
369ccd5b6